<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Passsword Generator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	
	background-color:#F8FCFB;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="PasswordGenerator_label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { 
	color: rgb(34, 34, 34);
	border-bottom-color: rgb(179, 200, 207);
}</string>
    </property>
    <property name="text">
     <string>Password Generator</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="CustomPass">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	rgb(34, 34, 34)
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>Customize your password</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>791</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color: #E7F4F1;
    border-radius: 10px;
    background-color:#E7F4F1;
    border: 2px solid #B3C8CF;
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>401</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	background-color:transparent;
}

</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QRadioButton" name="EasyToSayRB">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton::indicator {
   border-radius: 8px; 
    border: 2px solid #219C90;
    background-color: #ffffff;
}

QRadioButton::indicator:checked {
    border: 2px solid #219C90;
	background-color: #219C90;
}</string>
      </property>
      <property name="text">
       <string>Easy to say </string>
      </property>
     </widget>
     <widget class="QRadioButton" name="EasyToReadRB">
      <property name="geometry">
       <rect>
        <x>124</x>
        <y>10</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton::indicator {
   border-radius: 8px; 
    border: 2px solid #219C90;
    background-color: #ffffff;
}

QRadioButton::indicator:checked {
    border: 2px solid #219C90;
	background-color: #219C90;
}</string>
      </property>
      <property name="text">
       <string>Easy to read </string>
      </property>
     </widget>
     <widget class="QRadioButton" name="AllChaRB">
      <property name="geometry">
       <rect>
        <x>244</x>
        <y>10</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton::indicator {
   border-radius: 8px; 
    border: 2px solid #219C90;
    background-color: #ffffff;
}

QRadioButton::indicator:checked {
    border: 2px solid #219C90;
	background-color: #219C90;
}</string>
      </property>
      <property name="text">
       <string>All characters</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>401</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	
	background-color: transparent;
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QCheckBox" name="LowerCaseChB">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox::indicator {
    border: 2px solid #219C90;
    border-radius: 3px;
    background-color: #ffffff;
}

QCheckBox::indicator:checked {
	 background-color:  #219C90;
    border: 2px solid #219C90;
}

QCheckBox::indicator:checked:hover {
   
    border: 2px solid  #219C90;
}</string>
      </property>
      <property name="text">
       <string>Lowercase</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="UpperCaseChB">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox::indicator {
    border: 2px solid #219C90;
    border-radius: 3px;
    background-color: #ffffff;
}

QCheckBox::indicator:checked {
	 background-color:  #219C90;
    border: 2px solid #219C90;
}

QCheckBox::indicator:checked:hover {
   
    border: 2px solid  #219C90;
}</string>
      </property>
      <property name="text">
       <string>Uppercase</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="NumberChB">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox::indicator {
    border: 2px solid #219C90;
    border-radius: 3px;
    background-color: #ffffff;
}

QCheckBox::indicator:checked {
	 background-color:  #219C90;
    border: 2px solid #219C90;
}

QCheckBox::indicator:checked:hover {
   
    border: 2px solid  #219C90;
}</string>
      </property>
      <property name="text">
       <string>Numbers</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="SymbolChB">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox::indicator {
    border: 2px solid #219C90;
    border-radius: 3px;
    background-color: #ffffff;
}

QCheckBox::indicator:checked {
	 background-color:  #219C90;
    border: 2px solid #219C90;
}

QCheckBox::indicator:checked:hover {
   
    border: 2px solid  #219C90;
}</string>
      </property>
      <property name="text">
       <string>Symbols</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>10</y>
       <width>311</width>
       <height>61</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Password Length&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	
	background-color:#E7F4F1;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QSpinBox {
    border: 2px solid #B3C8CF;
    border-radius: 10px;
    padding: 8px;
    padding-left: 10px;
    padding-right: 20px;
    background-color: #ffffff;
    font-size: 14px;
    color: #333333;
    selection-background-color: #0078d7;
    selection-color: white;
}

QSpinBox:hover {
    border: 2px solid #B3C8CF;
}

QSpinBox:focus {
    border: 2px solid #B3C8CF;
    background-color: #f0f7ff;
}

QSpinBox::up-button, QSpinBox::down-button {
    border: none;
    background-color: transparent;
    width: 24px;
}

QSpinBox::up-button {
    subcontrol-origin: border;
    subcontrol-position: top right;
    border-left: 1px solid #B3C8CF;
    border-bottom: 1px solid #B3C8CF;
    border-top-right-radius: 19px;
}

QSpinBox::down-button {
    subcontrol-origin: border;
    subcontrol-position: bottom right;
    border-left: 1px solid #B3C8CF;
    border-top: 1px solid #B3C8CF;
    border-bottom-right-radius: 19px;
}</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QSlider::groove:horizontal {
    border: 1px solid #B3C8CF;      /* خط اصلی */
    height: 6px;                    /* ضخامت خط */
    background: #E7F4F1;            /* رنگ پس‌زمینه خط */
    border-radius: 3px;             /* گرد کردن خط */
}

QSlider::handle:horizontal {
    background: #219C90;            /* رنگ دایره */
    width: 18px;                    /* قطر دایره */
    height: 18px;
    margin: -7px 0;                 /* قرارگیری دقیق روی خط */
    border-radius: 9px;             /* نصف width = دایره کامل */
}
</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="GenerateButton">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>80</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-radius: 15px;
    background-color: #219C90;
    border: 2px solid #1A7F75;
}

QPushButton {
    color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>Generate</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../GUI/PyQt5/images/icons/bonus/icons-24/gear.png</normaloff>../GUI/PyQt5/images/icons/bonus/icons-24/gear.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="Password_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>250</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color:#D3E7E3;
    border-radius: 10px;
    background-color: #D3E7E3;
    border: 2px solid #cccccc;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Password_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>310</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color: #D3E7E3;
    border-radius: 10px;
    background-color:#D3E7E3;
    border: 2px solid #cccccc;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Copy_1">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>250</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color:#ffffff;
    border-radius: 10px;
    background-color: #ffffff;
    border: 2px solid #cccccc;
}
QPushButton:hover {
    border: 2px solid #219C90;
}

QPushButton:pressed {
    border: 2px solid #219C90;
    box-shadow: 0 0 10px rgba(52, 152, 219, 0.8);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/mahrayan/Desktop/Screenshot_30-8-2025_23329_passgenerator.com.jpeg</normaloff>C:/Users/mahrayan/Desktop/Screenshot_30-8-2025_23329_passgenerator.com.jpeg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>41</width>
      <height>41</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="Password_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>310</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color: #D3E7E3;
    border-radius: 10px;
    background-color: #D3E7E3;
    border: 2px solid #cccccc;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Password_1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color:#D3E7E3;
    border-radius: 10px;
    background-color:#D3E7E3;
    border: 2px solid #cccccc;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Copy_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>310</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color:#ffffff;
    border-radius: 10px;
    background-color: #ffffff;
    border: 2px solid #cccccc;
}
QPushButton:hover {
    border: 2px solid #219C90;
}

QPushButton:pressed {
    border: 2px solid #219C90;
    box-shadow: 0 0 10px rgba(52, 152, 219, 0.8);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/mahrayan/Desktop/Screenshot_30-8-2025_23329_passgenerator.com.jpeg</normaloff>C:/Users/mahrayan/Desktop/Screenshot_30-8-2025_23329_passgenerator.com.jpeg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>41</width>
      <height>41</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Copy_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>250</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color:#ffffff;
    border-radius: 10px;
    background-color: #ffffff;
    border: 2px solid #cccccc;
}
QPushButton:hover {
    border: 2px solid #219C90;
}

QPushButton:pressed {
    border: 2px solid #219C90;
    box-shadow: 0 0 10px rgba(52, 152, 219, 0.8);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/mahrayan/Desktop/Screenshot_30-8-2025_23329_passgenerator.com.jpeg</normaloff>C:/Users/mahrayan/Desktop/Screenshot_30-8-2025_23329_passgenerator.com.jpeg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>41</width>
      <height>41</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Copy_4">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>310</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color:#ffffff;
    border-radius: 10px;
    background-color: #ffffff;
    border: 2px solid #cccccc;
}
QPushButton:hover {
    border: 2px solid #219C90;
}

QPushButton:pressed {
    border: 2px solid #219C90;
    box-shadow: 0 0 10px rgba(52, 152, 219, 0.8);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/mahrayan/Desktop/Screenshot_30-8-2025_23329_passgenerator.com.jpeg</normaloff>C:/Users/mahrayan/Desktop/Screenshot_30-8-2025_23329_passgenerator.com.jpeg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>41</width>
      <height>41</height>
     </size>
    </property>
   </widget>
   <zorder>PasswordGenerator_label</zorder>
   <zorder>CustomPass</zorder>
   <zorder>frame</zorder>
   <zorder>Password_2</zorder>
   <zorder>Password_4</zorder>
   <zorder>Password_3</zorder>
   <zorder>Password_1</zorder>
   <zorder>Copy_1</zorder>
   <zorder>Copy_3</zorder>
   <zorder>Copy_2</zorder>
   <zorder>Copy_4</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>831</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
